snuba/migrations/snuba_migrations/sessions/0001_sessions.py

```diff
@@ -0,0 +1,149 @@
+                storage_set=StorageSetKey.SESSIONS,
+                table_name="sessions_hourly_local",
+                columns=aggregate_columns,
+                engine=table_engines.AggregatingMergeTree(
+                    order_by="(org_id, project_id, release, environment, started)",
+                    partition_by="(toMonday(started))",
+                    settings={"index_granularity": "256"},
+                ),
+            ),
+            operations.CreateMaterializedView(
+                storage_set=StorageSetKey.SESSIONS,
+                view_name="sessions_hourly_mv_local",
+                destination_table_name="sessions_hourly_local",
+                columns=aggregate_columns,
+                query=f"""
+                    SELECT
+                        org_id,
+                        project_id,
+                        toStartOfHour(started) as started,
+                        release,
+                        environment,
+                        quantilesIfState(0.5, 0.9)(
+                            duration,
+                            duration <> {MAX_UINT32} AND status == 1
+                        ) as duration_quantiles,
+                        countIfState(session_id, seq == 0) as sessions,
+                        uniqIfState(distinct_id, distinct_id != '{NIL_UUID}') as users,
+                        countIfState(session_id, status == 2) as sessions_crashed,
+                        countIfState(session_id, status == 3) as sessions_abnormal,
+                        uniqIfState(session_id, errors > 0) as sessions_errored,
+                        uniqIfState(distinct_id, status == 2) as users_crashed,
+                        uniqIfState(distinct_id, status == 3) as users_abnormal,
+                        uniqIfState(distinct_id, errors > 0) as users_errored
+                    FROM
+                        sessions_raw_local
+                    GROUP BY
+                        org_id, project_id, started, release, environment
+                """,
+            ),
+        ]
+
+    def backwards_local(self) -> Sequence[operations.Operation]:
+        return [
+            operations.DropTable(
+                storage_set=StorageSetKey.SESSIONS, table_name="sessions_raw_local",
+            )
+        ]
+
+    def forwards_dist(self) -> Sequence[operations.Operation]:
+        return [
+            operations.CreateTable(
+                storage_set=StorageSetKey.SESSIONS,
+                table_name="sessions_raw_dist",
+                columns=raw_columns,
+                engine=table_engines.Distributed(
+                    local_table_name="sessions_raw_local",
+                    sharding_key=None,  # TODO: what was the sharding key?
+                ),
+            ),
+            operations.CreateTable(
+                storage_set=StorageSetKey.SESSIONS,
+                table_name="sessions_hourly_dist",
+                columns=aggregate_columns,  # TODO: needs verification
```

Comment:
@fpacifici Do you know if this was changed at all? or the same list of columns was used?

Reply:
This is the current schema.

```
CREATE TABLE default.sessions_hourly_dist 
(
 `org_id` UInt64, 
  `project_id` UInt64, 
  `started` DateTime, 
   `release` LowCardinality(String), 
  `environment` LowCardinality(String), 
  `duration_quantiles` AggregateFunction(quantilesIf(0.5, 0.9), UInt32, UInt8),
  `sessions` AggregateFunction(countIf, UUID, UInt8), 
  `users` AggregateFunction(uniqIf, UUID, UInt8), 
  `sessions_crashed` AggregateFunction(countIf, UUID, UInt8), 
  `sessions_abnormal` AggregateFunction(countIf, UUID, UInt8), 
  `sessions_errored` AggregateFunction(uniqIf, UUID, UInt8), 
  `users_crashed` AggregateFunction(uniqIf, UUID, UInt8), 
  `users_abnormal` AggregateFunction(uniqIf, UUID, UInt8), 
  `users_errored` AggregateFunction(uniqIf, UUID, UInt8)) 
ENGINE = Distributed(snuba_sessions, default, sessions_hourly_local, org_id)
```